contracts/proposals/IncentiveUpdateExecutor.sol

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.7.5;
+pragma abicoder v2;
+
+import {DataTypes} from '../utils/DataTypes.sol';
+import {IAaveEcosystemReserveController} from '../interfaces/IAaveEcosystemReserveController.sol';
+import {IAaveIncentivesController} from '../interfaces/IAaveIncentivesController.sol';
+import {IATokenDetailed} from '../interfaces/IATokenDetailed.sol';
+import {ILendingPoolConfigurator} from '../interfaces/ILendingPoolConfigurator.sol';
+import {ILendingPoolData} from '../interfaces/ILendingPoolData.sol';
+
+contract IncentiveUpdateExecutor {
+
+  address constant AAVE_TOKEN = 0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9;
+  address constant ECO_RESERVE_ADDRESS = 0x1E506cbb6721B83B1549fa1558332381Ffa61A93;
+  address constant LENDING_POOL = 0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9;
+  address constant POOL_CONFIGURATOR = 0x311Bb771e4F8952E6Da169b425E7e92d6Ac45756;
+  address constant INCENTIVES_CONTROLLER_PROXY_ADDRESS = 0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5;
+
+  uint256 constant DISTRIBUTION_DURATION = 7776000;   // 90 days
+  uint256 constant DISTRIBUTION_AMOUNT = 138600 ether;
+  
+  function execute(IATokenDetailed[] memory _aTokens, IATokenDetailed[] memory _debtTokens) external {
```

Comment:
@ncitron not strong position, but I'm more for not passing any parameter, and hardcoding the addresses of aTokens/debtTokens on the payload, like here https://etherscan.io/address/0xC5Dc99a421e3FB9d64c7B5BBb47a5e072370eDE8#code
In my opinion is better for transparency for others to review that the parameters are correct, as checking the calldata values on the proposal creation tx is a bit more painful.

Reply:
Definitely agree. I Will hardcode these values once I have done the deployments.